frontend/src/metabase/dashboard/reducers.js

```diff
@@ -160,7 +160,7 @@ const dashcardData = handleActions(
       next: (state, { payload: { clearCache = true } = {} }) =>
         clearCache ? {} : state,
     },
-    [FETCH_CARD_DATA]: {
+    [fetchCardDataAction.fulfilled]: {
```

Comment:
this is really brave I hope you'll not break anything 😄

Reply:
I've done it plenty of times already 😂